src/Spout/Reader/CSV/RowIterator.php

```diff
@@ -148,23 +144,36 @@ public function next()
 
     /**
      * Returns the next line, converted if necessary to UTF-8.
-     * Neither fgets nor fgetcsv don't work with non UTF-8 data... so we need to do some things manually.
+     * As fgetcsv() does not manage correctly encoding for non UTF-8 data,
+     * we remove manually whitespace with ltrim or rtrim (depending on the order of the bytes)
      *
      * @return string|false The next line for the current file pointer, encoded in UTF-8 or FALSE if nothing to read
      * @throws \Box\Spout\Common\Exception\EncodingConversionException If unable to convert data to UTF-8
      */
     protected function getNextUTF8EncodedLine()
```

Comment:
rename this function `getNextUTF8EncodedRow` as well as all mentions of "line" by "row" since fgetcsv returns an array.